src/frontend/cypress/e2e/landing-page/landingpage.cy.ts

```diff
@@ -0,0 +1,45 @@
+describe('Test DPIA Landing Page', () => {
+  beforeEach(() => {
+    // Cypress starts out with a blank slate for each test
+    // so we must tell it to visit our website with the `cy.visit()` command.
+    // Since we want to visit the same URL at the start of all our tests,
+    // we include qit in our beforeEach function so that it runs before each test
+    let url = Cypress.config().baseUrl; //accessing baseUrl
+    // this line is by pass a ts compile error for visit function
+    // TS2769: No overload matches this call
+    if (!url) url = 'http://localhost:8080';
```

Comment:
do we need this line if we are already specifying default in the `cypress.config.ts` file? 🤔

Reply:
if not add this line, can not build successful, TS will throw error
```
cypress/e2e/landing-page/landingpage.cy.ts:9:14 - error TS2769: No overload matches this call.
  Overload 1 of 2, '(url: string, options?: Partial<VisitOptions> | undefined): Chainable<AUTWindow>', gave the following error.
    Argument of type 'string | null' is not assignable to parameter of type 'string'.
      Type 'null' is not assignable to type 'string'.
  Overload 2 of 2, '(options: Partial<VisitOptions> & { url: string; }): Chainable<AUTWindow>', gave the following error.
    Argument of type 'string | null' is not assignable to parameter of type 'Partial<VisitOptions> & { url: string; }'.
      Type 'null' is not assignable to type 'Partial<VisitOptions> & { url: string; }'.
        Type 'null' is not assignable to type 'Partial<VisitOptions>'.

9     cy.visit(url);
```